test/tests/functional/pbs_job_array_comment.py

```diff
@@ -68,6 +68,11 @@ def test_job_array_comment(self):
         attr = {'event': 'execjob_begin', 'enabled': 'True'}
         self.server.create_import_hook(hook_name, attr, hook_body)
 
+        # Check if the hook copy was successful
+        self.server.log_match(".*successfully sent hook file.*" +
+                              hook_name + ".PY" + ".*", regexp=True,
+                              max_attempts=100, interval=5)
```

Comment:
Two things here.  First, you don't need the ".*" at the end.  log_match() does substring matches.  Second, 500 seconds is longer than the timeout value for the test.  Why don't you change this to offset=5, interval=2, and leave max_attempts at the default of 60.  This means the test will fail if the hook doesn't get sent to the mom, rather than timeout.

Reply:
Thanks Bhroam. I have made the suggested changes.